### PR TITLE
Bump up spotbugs

### PR DESCRIPTION
**Pull request by KengoTODA**
This PR releases v0.2.2 that depends on SpotBugs 3.1.9

### CODE REVIEW

**Comment by KengoTODA** [comment]
@spotbugs/core-devs please review this change, thanks in advance!

File: mvnw [hunk 1]
```diff
@@ -200,65 +200,6 @@ if [ -z "$BASE_DIR" ]; then
   exit 1;
 fi
 
-##########################################################################################
```
Comment (ThrawnCA) [inline mvnw]: Why are we changing the Maven wrapper?

Comment (KengoTODA) [reply mvnw]: To provide latest solutions to users. Not also maven but also maven-plugins and SpotBugs are kept updated in this project.

**Comment by KengoTODA** [comment]
Thanks! I'll release v0.2.2 later.

**Pull request merged**